pallets/committee/src/lib.rs

```diff
@@ -125,6 +126,11 @@ pub mod pallet {
 	pub type Votes<T: Config> =
 		StorageMap<_, Blake2_128Concat, HashFor<T>, VoteAggregate<AccountIdFor<T>, BlockNumberFor<T>>, OptionQuery>;
 
+	/// Storte a mapping (hash) -> BlockNumber for new members' voting timeouts
```

Comment:
```suggestion
	/// Stores the block height at which point a member is eligible to cast their vote
	/// For new members, this will be the block they were added as members plus the duration of one voting period.
	/// This means new members are eligible to cast their vote in the next voting period.
```